docs/source/tutorials/using_text.rst

```diff
@@ -0,0 +1,480 @@
+`مرحبا بالعالم`.
+
+Here is a simple Hello World animation.
+
+.. manim:: HelloWorld 
+    :save_last_frame:
+    :ref_classes: Text
+
+    class HelloWorld(Scene):
+        def construct(self):
+            text = Text("Hello world").scale(3)
+            self.add(text)
+
+You can also use :class:`~.MarkupText` where PangoMarkup (:class:`~.MarkupText`)
+can be used to render text. For example:
+
+.. manim:: SingleLineColor 
+    :save_last_frame:
+    :ref_classes: MarkupText
+
+    class SingleLineColor(Scene):
+        def construct(self):
+            text = MarkupText(f'all in red <span fgcolor="{YELLOW}">except this</span>', color=RED)
+            self.add(text)
+
+.. _Pango library: https://pango.gnome.org
+
+Working with :class:`~.Text`
+============================
+
+This section explains the properties of :class:`~.Text` and how can it be used
+in your animations.
+
+Using Fonts
+-----------
+
+You can set a different font using :attr:`~.Text.font`.
+
+.. note:: 
+
+    The font used must be installed in your system, and Pango should know
+    about it. You can get a list of fonts using :func:`manimpango.list_fonts`.
+
+    >>> import manimpango
+    >>> manimpango.list_fonts()
+    [...]
+
+
+.. manim:: FontsExample 
+    :save_last_frame:
+    
+    class FontsExample(Scene):
+        def construct(self):
+            ft = Text("Noto Sans", font="Noto Sans")
+            self.add(ft)
+
+Setting Slant and Weight
+------------------------
+Slant is the style of the Text, and it can be ``NORMAL`` (the default), 
+``ITALIC``, ``OBLIQUE``. Usually, for many fonts both ``ITALIC`` and
+``OBLIQUE`` looks similar, but ``ITALIC`` uses **Roman Style**, which 
+``OBLIQUE`` uses **Italic Style**.
+
+Weight specifies the boldness of a font. You can see a list in
+:class:`manimpango.Weight`.
+
+.. manim:: SlantsExample
+    :save_last_frame:
+
+    class SlantsExample(Scene):
+        def construct(self):
+            a = Text("Italic", slant=ITALIC)
+            self.add(a)
+
+.. manim:: DifferentWeight 
+    :save_last_frame:
+
+    class DifferentWeight(Scene):
+        def construct(self):
+            import manimpango
+
+            g = VGroup()
+            weight_list = dict(sorted({weight: manimpango.Weight(weight).value for weight in manimpango.Weight}.items(), key=lambda x: x[1]))
+            for weight in weight_list:
+                g += Text(weight.name, weight=weight.name, font="Open Sans")
+            self.add(g.arrange(DOWN).scale(0.5))
+
+Using Colors
+------------
+
+You can set the color of the text using :attr:`~.Text.color`.
+
+For example,
+
+.. manim:: SimpleColor
+    :save_last_frame:
+
+    class SimpleColor(Scene):
+        def construct(self):
+            col = Text("RED COLOR", color=RED)
+            self.add(col)
+
+You can use utilities like :attr:`~.Text.t2c` for coloring specific characters 
+from others. This may be problematic if your text contains ligatures
+as explained in :ref:`iterating-text`.
```

Comment:
```suggestion
You can use utilities like :attr:`~.Text.t2c` for coloring specific characters.
This may be problematic if your text contains ligatures
as explained in :ref:`iterating-text`.
```